### ISSUE TITLE
dialogue script error

### ISSUE THREAD

**Issue by Gawidev** [Tenome/Gurumin-Modding#3]
**Describe the bug**
Same script error as last time. Occurs during the second "Vanishing cakes" dialogue with Fan.

```
Script error occurred (UPIPE)
Number of arguments is incorrect. (9794)
```

**Screenshots**
<img src="https://github.com/user-attachments/assets/f563b456-1b58-4988-9bb1-1a52b7d300c6" width="50%" height="50%">
<img src="https://github.com/user-attachments/assets/f798257b-ea77-48a7-a2ed-6cfb98fd40dd" width="50%" height="50%">

**Save file**
Added new saves, the one of interest should be gdata211.sav
https://mega.nz/folder/5Ts3hY4R#zN8lyBRxMRv8kGLlV6tT-Q

**Comment by Tenome** [comment]
Expected as much. If it's the same issue then I'll apply the fix to all the bubbles I added.

This time, I fixed it by adding an extra `"`, which is the opposite of what I did with Doug... I should try and find out why exactly that is. Guess I'll look through a C tutorial online at some point.

Remove the `.txt` from the filename and drop this in `Gurumin - A Monstrous Adventure\bin\`.

[map.itm.txt](https://github.com/user-attachments/files/17552014/map.itm.txt)
